### PR TITLE
Fixes markdown generic code styles

### PR DESCRIPTION
**Pull request by micmmakarov**

### CODE REVIEW

**Comment by micmmakarov** [comment]
@gkop ^

**Comment by gkop** [comment]
+1

**Comment by davekapp** [comment]
wfm :+1:

File: app/assets/stylesheets/posts.css.scss [hunk 1]
```diff
@@ -111,6 +111,11 @@
     padding-right: 0px;
   }
 
+  h4 {
+    font-size: 24px !important;
```
Comment (micmmakarov) [inline app/assets/stylesheets/posts.css.scss]: doing it in the same style as it is below

**Comment by micmmakarov** [comment]
@gkop some more changes added ^
didn't do a separate PR

**Pull request merged**